### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Stihdjia**
Two new font graphic files

### CODE REVIEW

**Comment by AquariusPower** [comment]
nice!
I suggest this subject: New text fonts

**Comment by AquariusPower** [comment]
very nice! the large (wide) font looks really nice, thx!
I created this pull request to patch your branch allowing it be a  user config option :)
https://github.com/Stihdjia/ivan/pull/1

File: Main/Source/igraph.cpp [hunk 1]
```diff
@@ -86,6 +86,12 @@ void igraph::Init()
     graphics::SetSwitchModeHandler(ivanconfig::SwitchModeHandler);
 #endif
     graphics::LoadDefaultFont(game::GetDataDir() + "Graphics/Font.png");
```
Comment (AquariusPower) [inline Main/Source/igraph.cpp]: here could test ==1 to avoid double loading the png files

File: Main/Source/iconf.cpp [hunk 2]
```diff
@@ -669,6 +642,9 @@ void ivanconfig::ShowItemsAtPlayerSquareChanger(cycleoption* O, long What)
 {
 //  if(What==1 && GetStartingDungeonGfxScale()<3){ //above head asked
 //    What=2; //upgrade to 1st corner
+//  }
+//  if(What==1 && GetStartingFontGfx()<3){ //I have no idea what these lines mean, copying font stuff in case it's ever important --Stihdjia
```
Comment (AquariusPower) [inline Main/Source/iconf.cpp]: this block of comments/deadCode can be safely removed (is just some stuff I left behind and forgot to clean), 
the automatic corners calculation is elsewhere now

**Comment by Stihdjia** [comment]
Thanks for the tips man. I'm not sure whether you get this message, I'm
just replying to an email I got from Github. I've just about got the
repaired files ready, testing now.

**Comment by AquariusPower** [comment]
cool thx! 
btw that link is correct, 
https://github.com/Stihdjia/ivan/compare/master...Attnam:master
I just tested equivalent to it as some new PRs were merged right now.
it is to merge using the browser, I use chromium/chrome.

to see the things that went missing I looked at the "Files Changed" tab on the browser and did the review thru there too :)

File: Main/Source/iconf.cpp [hunk 3]
```diff
@@ -680,43 +656,6 @@ void ivanconfig::WindowWidthChanger(numberoption* O, long What)
   O->Value = What;
 }
 
-void ivanconfig::SelectedBkgColorChanger(stringoption* O, cfestring& What)
```
Comment (AquariusPower) [inline Main/Source/iconf.cpp]: here, this is one of the blocks (the biggest one) that went missing

**Comment by Stihdjia** [comment]
I think I've got everything updated. I was using the source from 0.53,
didn't realize master version had moved ahead.

**Comment by Stihdjia** [comment]
Aw dang I missed a new master again I think

**Comment by fejoa** [comment]
> Aw dang I missed a new master again I think

Not your fault, there happens to be a merge frenzy going on :)

**Comment by Stihdjia** [comment]
Well I'll leave it to you, the masters. It looks like you're managing to
incorporate my stuff in spite of my utter confusion with Github and
programming in general.

The newer stuff I added attempts to address the things Aquarius mentioned
but misses you guys' updates. Glad I could contribute a little something!

**Comment by AquariusPower** [comment]
better let this "round" of mergings end, and then we can update your master with newest Attnam:master, and work from that on :)

**Comment by fejoa** [comment]
>better let this "round" of mergings end, and then we can update your master with newest Attnam:master, and work from that on :)

Ok I'm taking a break again :)

**Comment by Stihdjia** [comment]
Well I'm completely lost now, I think I might close this pull request and try again. But maybe not til tomorrow. I just now figured out how to get back to these comments instead of emailing my replies to GitHub.

**Comment by AquariusPower** [comment]
I think the easiest way would be that you create a new branch based on Attnam:master named like NewFonts, instead of using your master branch, then just add your new code on it, and create the new PR based on it :)

**Comment by AquariusPower** [comment]
@Stihdjia 
as I didnt know you had c++ code ready for the user option, I ended creating this branch: https://github.com/AquariusPower/ivan/tree/NewFontsByStihdjia
The code there is quite similar to yours and contains your commit with the Font*.png files.
I just updated it to latest Attnam:master if it helps.
But creating your own branch would still be a good exercise :)

**Comment by AquariusPower** [comment]
@fejoa 
> Ok I'm taking a break again :)

but the more mergings happen the easier is to maintain remaining branches :)
(I just need to hold my hand b4 adding "ready" on subjects xD)

**Pull request closed without merging**